settings/dev.settings.json

```diff
@@ -2,9 +2,9 @@
   "ROOT_URL": "",
   "MAIL_URL": "",
   "reaction": {
-    "REACTION_USER": "",
-    "REACTION_AUTH": "",
-    "REACTION_EMAIL": ""
+    "REACTION_USER": "admin",
```

Comment:
It didn't. That's why we see the file in the PR. The `dev.settings.json` file is meant to be an example file and always left blank.